troubleshooting.md

```diff
@@ -69,3 +69,11 @@ communicate with a registry and not use tls verification.
   * Turn off tls verification by passing false to the tls-verification option.
   * I.e. `podman push --tls-verify=false alpine docker://localhost:5000/myalpine:latest`
 ---
+### 4) rootless containers cannot ping hosts
+
```

Comment:
It would be nice to have a little verbiage here explaining the issue, an example of the symptom, then the solution.  Perhaps:
```
When using the ping command from a non-root container, the command may fail.

#### Symptom

console 
(example ping failure)


#### Solution

It is most likely necessary to enable unprivileged pings on the host.  Verify that the UID of the user is part of the range in the `/proc/sys/net/ipv4/ping_group_range` file of the host.  

To persist the change, use the `sysctl` command.  For example:  
`sysctl -w "net.ipv4.ping_group_range=0 2000000"`.

Reply:
good idea.  Amended in the new version